charts/piraeus/templates/operator-controller.yaml

```diff
@@ -4,6 +4,16 @@ kind: LinstorController
 metadata:
   name: {{ template "operator.fullname" . }}-cs
   namespace: {{ .Release.Namespace }}
+  labels:
+    app.kubernetes.io/name: {{ template "operator.fullname" . }}-operator-controller
```

Comment:
should be the same as the `name` field I think, so `{{ template "operator.fullname" . }}-cs`